services/QuillLMS/client/app/bundles/Teacher/components/accounts/new/onboarding_tour/__tests__/introduction_slides.test.tsx

```diff
@@ -29,37 +23,16 @@ describe('introductionSlides', () => {
     });
     expect(buttons).toHaveLength(4);
 
-    await userEvent.click(buttons[1]);
-    expect(setTour).toHaveBeenCalledWith(ENGLISH_LEARNERS);
-
-    await userEvent.click(buttons[3]);
-    expect(setTour).toHaveBeenCalledWith(HIGH_SCHOOL_SOCIAL_STUDIES);
+    await userEvent.click(buttons[0]);
+    expect(setTour).toHaveBeenCalledWith(ELEMENTARY_AND_MIDDLE_SCHOOL_ELA);
   });
 
   describe('snapshots for each slide', () => {
-    test('slide 0 matches snapshot', () => {
-      const { container } = render(slides[0]);
-      expect(container.firstChild).toMatchSnapshot();
-    });
-
-    test('slide 1 matches snapshot', () => {
-      const { container } = render(slides[1]);
-      expect(container.firstChild).toMatchSnapshot();
-    });
-
-    test('slide 2 matches snapshot', () => {
-      const { container } = render(slides[2]);
-      expect(container.firstChild).toMatchSnapshot();
-    });
-
-    test('slide 3 matches snapshot', () => {
-      const { container } = render(slides[3]);
-      expect(container.firstChild).toMatchSnapshot();
-    });
-
-    test('slide 4 matches snapshot', () => {
-      const { container } = render(slides[4]);
-      expect(container.firstChild).toMatchSnapshot();
-    });
+    slides.forEach((slide, index) => {
```

Comment:
I like this iterating test since it should cover us if we add new slides or shuffle them around later.